Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1008,8 +1008,8 @@ def plot2D(self,data1,data2,tmpl,gm,vtk_backend_grid=None,vtk_backend_geo=None):
           #png.SetFileName("/git/uvcdat/Packages/vcs/Share/uvcdat_texture.png")
           #T=vtk.vtkTexture()
           #T.SetInputConnection(png.GetOutputPort())
-          if isinstance(gm,isofill.Gfi):
-              mappers.append([mapper,])
+          #if isinstance(gm,isofill.Gfi):
```

Comment:
@aashish24 did you run ctest? I was talking about the commented out lines above. I'm pretty sure these lines are needed. Othwerwise your mappers will be empty and nothing would be drawn? no?